test/Dialect/FIRRTL/check-comb-cycles.mlir

```diff
@@ -587,6 +589,18 @@ firrtl.circuit "CycleThroughForceable"   {
 
 // -----
 
+firrtl.circuit "CycleThroughForceableRef"   {
```

Comment:
Thanks for including these, and for creating MLIR tests for the 5462 specifically!